src/h3lib/include/polygon.h

```diff
@@ -40,18 +40,38 @@
 /** Macro: Whether a GeoLoop is empty */
 #define IS_EMPTY_GEOFENCE(geoloop) geoloop->numVerts == 0
 
+/**
+ * Values representing polyfill containment modes, to be used in
+ * the `flags` bit field.
+ */
+typedef enum {
+    CENTER_CONTAINMENT = 0,
+    FULL_CONTAINMENT = 1,
+    OVERLAPPING = 2,
+    INVALID_CONTAINMENT = 3
+} ContainmentMode;
```

Comment:
we may want to prefix the enun names with something here (`POLYGON_CENTER_CONTAINMENT`?) so that it is clear a valid enum is being fed into `polygonToCells`.

Reply:
That makes sense - leaning towards `CONTAINMENT_` as the prefix, so we'd have

- `CONTAINMENT_CENTER`
- `CONTAINMENT_FULL`
- `CONTAINMENT_OVERLAPPING`
- `CONTAINMENT_INVALID`

One thought here is that we might apply the same modes to [circles](https://github.com/uber/h3/issues/489) or bounding boxes at some point, so I don't necessarily think we want to bake `POLYGON` into the name.